src/chat/components/bubble/CustomVideoBubble.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useRef, useState } from 'react';
+import { StyleSheet, View, Image, TouchableOpacity } from 'react-native';
+import Video, { VideoRef } from 'react-native-video';
+import { CustomImageVideoBubbleProps } from './CustomImageBubble';
+
+export const CustomVideoBubble: React.FC<CustomImageVideoBubbleProps> = ({
```

Comment:
should we create new CustomVideoBubbleProps in this file?

Reply:
updated